core/run_ctest/run_single_ctest.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/python3
+import sys, time
+
+from program_input import p_input
+from main import test_conf_file
+from parse_input import *
+
+run_mode = p_input["run_mode"]
+project = p_input["project"]
+mapping = parse_mapping(p_input["mapping_path"])
+from run_test import run_test_batch
+
+def main(argv):
+    print(">>>>[ctest_core] running project {}".format(project))
+    s = time.time()
+    ctestname = argv[1]
+    if run_mode == "run_ctest":
```

Comment:
Minor: Not sure if this is needed, because we're having a special python file to run a single test. Adding this will require us to modify run_mode in program_input each time we switch between batch mode and single test mode, which can be avoided.